From 34d84c3b0f63978efcb5136a6fd2e7e9d889da70 Mon Sep 17 00:00:00 2001
From: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Date: Sun, 5 Mar 2023 13:24:28 +0100
Subject: [PATCH] dt-bindings: soc: qcom: smd-rpm: re-add missing qcom,rpm-msm8994
Git-commit: 34d84c3b0f63978efcb5136a6fd2e7e9d889da70
Patch-mainline: v6.4-rc1
References: git-fixes

Re-add the qcom,rpm-msm8994 compatible, dropped during conversion from
TXT to DT schema:

  apq8094-sony-xperia-kitakami-karin_windy.dtb: smd: rpm:rpm-requests:compatible:0: 'qcom,rpm-msm8994' is not one of ['qcom,rpm-apq8084' ...]

Fixes: f935a752f229 ("dt-bindings: soc: qcom: smd-rpm: Convert binding to YAML schema")
Signed-off-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Acked-by: Rob Herring <robh@kernel.org>
Reviewed-by: Petr Vorel <petr.vorel@gmail.com>
Reviewed-by: Konrad Dybcio <konrad.dybcio@linaro.org>
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
Link: https://lore.kernel.org/r/20230305122428.167580-1-krzysztof.kozlowski@linaro.org
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 Documentation/devicetree/bindings/soc/qcom/qcom,smd-rpm.yaml |    2 ++
 1 file changed, 2 insertions(+)

--- a/Documentation/devicetree/bindings/soc/qcom/qcom,smd-rpm.yaml
+++ b/Documentation/devicetree/bindings/soc/qcom/qcom,smd-rpm.yaml
@@ -38,6 +38,7 @@ properties:
       - qcom,rpm-msm8953
       - qcom,rpm-msm8974
       - qcom,rpm-msm8976
+      - qcom,rpm-msm8994
       - qcom,rpm-msm8996
       - qcom,rpm-msm8998
       - qcom,rpm-sdm660
@@ -59,6 +60,7 @@ if:
           - qcom,rpm-msm8916
           - qcom,rpm-msm8974
           - qcom,rpm-msm8953
+          - qcom,rpm-msm8994
 then:
   required:
     - qcom,smd-channels
